src/CLI.hs

```diff
@@ -67,7 +79,8 @@ data Command w
                     }
 
   | HEVM            { spec       :: w ::: String               <?> "Path to spec"
-                    , soljson    :: w ::: String               <?> "Path to .sol.json"
+                    , sol        :: w ::: String               <?> "Path to .sol"
```

Comment:
I think the UX of taking a soliidty file instead of some build output is nice, but I think it's going to make it harder to integrate with real world projects which will have their own build system. Probably the nice way to do this is to offer some additional options that let us pull contracts from foundry build output. I'm fine if this gets done in a separate PR.